session/session.go

```diff
@@ -2702,9 +2702,9 @@ var (
 	errResultIsEmpty = dbterror.ClassExecutor.NewStd(errno.ErrResultIsEmpty)
 	// DDLJobTables is a list of tables definitions used in concurrent DDL.
 	DDLJobTables = []string{
-		"create table tidb_ddl_job(job_id bigint not null, reorg int, schema_id bigint, table_id bigint, job_meta longblob, is_drop_schema int, primary key(job_id))",
```

Comment:
We remove `is_drop_schema` field, so do we need update `RFC`?

Reply:
yes, RFC will be updated